src/pages/Main.tsx

```diff
@@ -0,0 +1,110 @@
+import { Box } from '@mui/material'
+import MainMenu from '@/components/features/MainMenu'
+import ProfileMenu from '@/components/features/ProfileMenu'
+import useMediaQuery from '@mui/material/useMediaQuery'
+import SmallMainMenu from '@/components/features/SmallMainMenu'
+import ArrowBackIosNewIcon from '@mui/icons-material/ArrowBackIosNew'
+import ArrowForwardIosIcon from '@mui/icons-material/ArrowForwardIos'
+import LanguageSwitcher from '@/components/features/LanguageSwitcher'
+import { useState } from 'react'
+
+// import { Route, Routes } from "react-router-dom";
+// import { Vendors } from "../Vendors/Vendors";
+
+export default function Main() {
+  const isSmallScreen = useMediaQuery('(max-width: 800px)')
+  const [isHideMenu, setIsHideMenu] = useState(false)
+
+  function hideMenuHandler() {
+    setIsHideMenu(true)
+  }
+
+  function showMenuHandler() {
+    setIsHideMenu(false)
+  }
+
+  return (
+    <>
+      <Box
+        component="section"
+        sx={{
+          display: 'flex'
+        }}>
+        {isSmallScreen ? null : (
+          <Box component="div" sx={{ display: 'flex' }}>
+            <Box
+              component="article"
+              sx={{
+                display: 'flex'
+              }}>
+              {isHideMenu ? (
+                <ArrowForwardIosIcon
+                  onClick={showMenuHandler}
+                  sx={{
+                    alignSelf: 'center',
+                    height: '100vh'
+                  }}
+                />
+              ) : (
+                <>
+                  <MainMenu />
+                  <ArrowBackIosNewIcon onClick={hideMenuHandler} sx={{ alignSelf: 'center' }} />
+                </>
+              )}
+            </Box>
+          </Box>
+        )}
```

Comment:
I think this makes sense to be called `MainMenu` and your current `MainMenu` can be changed to `MenuLinkItems` or maybe you can call this one `MenuDrawer`. Think about the names but i believe this can be into its own component as its just a piece to be added to the layout.

Reply:
Fixed